frontend/src/vars.scss

```diff
@@ -129,7 +129,8 @@ $z_main_nav: 948;
 $z_pinned_dashboards_popup: 962;
 $z_mobile_nav_overlay: 931;
 $z_top_navigation: 800;
-$z_graph_tooltip: 740;
+$z_graph_tooltip: 951;
+$z_plugin_metrics_drawer: 950;
```

Comment:
Let's keep this file ordered otherwise why keep a list. Did you mean 750/751?

Reply:
Ah yup will order. And nope, I mean 950/1. That's needed for the drawer to display over the top bar and left nav